contracts/ValidatorSet/modules/Withdrawal/Withdrawal.sol

```diff
@@ -8,6 +8,9 @@ import "./../AccessControl/AccessControl.sol";
 import "./../../ValidatorSetBase.sol";
 import "./../../libs/WithdrawalQueue.sol";
 
+error NoWithdrawalAvailable();
```

Comment:
Custom errors that are module-specific to be declared in the module's interface or in the module itself if the interface is not available. (internal styling convention)

Reply:
Moved to the interface